Qt/mainwindow.cpp

```diff
@@ -141,7 +142,7 @@ void MainWindow::closeAct()
 void MainWindow::openmsAct()
 {
 	QString confighome = getenv("XDG_CONFIG_HOME");
-	QString memorystick = confighome + "/ppsspp/PSP";
+	QString memorystick = confighome + "/ppsspp";
```

Comment:
What's the motivation for this change?

Reply:
Just unification with behavior of Windows version. Besides I've found there's more folders in memstick than just PSP.
https://doodypsp.blogspot.com/2007/10/psps-directory-structure-explained_06.html

Reply:
Oh right, forgot that's how we define it there. Good one.